README.md

```diff
@@ -76,6 +78,37 @@ We recommend installing RDKit with deepchem if you use conda.
 conda install -y -c conda-forge rdkit
 ```
 
+### Latest version
+
+For using general utilites like Molnet, Featurisers, Datasets, etc, then, you install deepchem via pip.  

```

Comment:
Add the line "The nightly version is built by the HEAD of DeepChem" to start this section